pom.xml

```
@@ -6,7 +6,8 @@
 
     <groupId>io.numaproj.numaflow</groupId>
     <artifactId>numaflow-java</artifactId>
-    <version>0.0.0-SNAPSHOT</version>
+    <version>0.1.0-SNAPSHOT</version>
```

```suggestion
    <version>0.3.0-SNAPSHOT</version>
```

should it still be `SNAPSHOT`?

---

will publish a snapshot package first, and if everything is good, will publish a release version.